test/OrchardCore.Tests/OrchardCore.Tests.csproj

```diff
@@ -24,6 +23,7 @@
     <None Remove="Modules\OrchardCore.OpenId\RecipeFiles\app-recipe2.json" />
     <None Remove="Modules\OrchardCore.OpenId\RecipeFiles\app-recipe3.json" />
     <None Remove="Modules\OrchardCore.OpenId\RecipeFiles\scope-recipe.json" />
+    <None Remove="NLog.config" />
```

Comment:
I don't think we need many changes here, please look into https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

Reply:
Log templates are different, such as paths
```
     <target xsi:type="File" name="file"
             fileName="${var:configDir}/App_Data/logs/${orchard-tenant-name}/orchard-log-${shortdate}.log"
             layout="${longdate}|${orchard-tenant-name}|${aspnet-traceidentifier}|${event-properties:item=EventId}|${logger}|${uppercase:${level}}|${message} ${exception:format=ToString,StackTrace}"
     />
```

Reply:
if you don't split the file, it seems that you can only write to a single tenant's log